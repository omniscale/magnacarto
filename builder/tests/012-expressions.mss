@foo: 1;
@bar: 12+12;
@bar: -12;
@bar: -12*12;
@bar: 12/2;
@bar: @foo * 12;
@baz: 3 * ( 1 + 2 );
@foo: #ddd;
@bar: fadein(@foo, 12/2);
@bar: fadein(@foo, 12%);

#class {
    line-join: miter;
    text-face-name: "foo", "bar";
}

@fonts: "foo", "bar";