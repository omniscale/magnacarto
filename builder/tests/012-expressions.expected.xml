<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="class" filter-mode="first">
    <Rule></Rule>
  </Style>
  <Layer name="class" srs="" status="off">
    <StyleName>class</StyleName>
  </Layer>
</Map>